### ISSUE TITLE
Improvement on the "Auto rename by Title"

### ISSUE THREAD

**Issue by DesBw** [glushchenko/fsnotes#1775]
This is a very useful feature. I appreciate it you have it here. But, it needs a little tweak so that FSnotes will play good with other apps such as Obsidian. Currently, if we select "Auto renmame by title" in setting, editing notes that are already created  in Obisidian or other app will get renamed to the first line text. That is too much. I think a better option would be to not touch the names of old notes; and rename only the ones we are creating inside fsnotes.

**Comment by glushchenko** [comment]
Files created in FSNotes are no different from any other markdown files. How to implement this?

**Comment by DesBw** [comment]
>After a while, users will come to me asking "why aren't the notes renamed"?
I don't think that ever happens because if a user editing an old file, he/she already have file name assigned to it. In rare occasions that we need to rename old file, it is not that hard to hit CMD+R. 

The problem with the current system is any file we edit inside FSNotes is going to renamed. It could cause a lot of unexpected results. 

As to the implementation, how about using the CreationDate attribute provided by the macOS?
Auto rename a  file --> iff it has been created in the last few seconds (or sth like that).

**Comment by DesBw** [comment]
Look at a comment by @xor-gate [here](https://github.com/glushchenko/fsnotes/issues/1222) 

> I think this feature works, but it did automagicly rename my files which I did not expect. If I use the setting other than "Auto Rename By Title" it should not touch my filenames every again right? I'm now using the "Format: Untitled Note" setting.

**Comment by glushchenko** [comment]
> As to the implementation, how about using the CreationDate attribute provided by the macOS?

That's possible, I can even make an additional option for that.

**Comment by DesBw** [comment]
> That's possible, I can even make an additional option for that.
thank you

**Comment by DesBw** [comment]
Wow, that is impressive man.  
Yes, it is working. 
May I suggest the name "Autoname by Title", because this one doesn't rename existing files. It assigns names to the newly created ones. 

Brilliant. 
Thank you.

**Comment by DesBw** [comment]
Unfortunately, the modification  is not working. Here, I have recorded a video to show that the setting is not applying. 
https://monosnap.com/file/30OPWxCZI6q95M0CFHviTbHk1YdsIk
The video also shows that the Fsnotes is not respecting the YAML to rename the file.

**Comment by glushchenko** [comment]
@DesBw please check this:

https://github.com/glushchenko/fsnotes/releases/download/6.9.8/FSNotes_6.9.8.zip

**Comment by DesBw** [comment]
I am afraid I am getting the same behavior (renaming files based on the first line). 
https://monosnap.com/file/RhselaliDVG54PSIiRYQhBrgW1Go9M

**Comment by glushchenko** [comment]
This works through checking the file creation date, I don't see any bugs here.

```
    func isOlderThanOneDay(from date: Date? = nil) -> Bool {
        if let date = date, let differenceInDays = Calendar.current.dateComponents([.day], from: date, to: Date()).day {
            return differenceInDays >= 1
        }
        
        return false
    }
```

**Comment by glushchenko** [comment]
@RainofTerra 100%

**Comment by glushchenko** [comment]
Implemented in 6.10.0 and available in AppStore
